app/presenters/transaction_file_tail.presenter.js

```diff
@@ -22,8 +22,8 @@ class TransactionFileTailPresenter extends BasePresenter {
       col01: 'T',
       col02: this._leftPadZeroes(data.index, 7),
       col03: this._numberOfLinesInFile(data.index),
-      col04: data.invoiceTotal,
-      col05: data.creditTotal
+      col04: data.invoiceValue,
+      col05: this._signedCreditValue(data.creditNoteValue, true)
```

Comment:
If my previous suggestion is taken on this would become

```suggestion
      col05: this._reverseSignOfValue(data.creditNoteValue)
```